scout/server/blueprints/institutes/controllers.py

```diff
@@ -279,18 +279,16 @@ def get_sanger_unevaluated(store, institute_id, user_id):
     return unevaluated
 
 
-def gene_variants(store, variants_query, institute_id, page=1, per_page=50):
+def gene_variants(store, pymongo_cursor, institute_id, page=1, per_page=50):
     """Pre-process list of variants."""
 
-    # We need to call variants_collection.count_documents here
-    variant_count = variants_query.count()
+    variant_count = count_cursor(pymongo_cursor)
```

Comment:
Clever solution! Hope you have checked that the cursor can be traversed multiple times, just so it does not work as a generator which gets exhausted after one loop-through